rs/tests/src/rosetta_tests/governance_client.rs

```diff
@@ -10,17 +9,25 @@ use ic_nns_governance_api::{
     },
     proposal_submission_helpers::create_make_proposal_payload,
 };
+use ic_registry_subnet_type::SubnetType;
 use ic_system_test_driver::{
     driver::{
         test_env::TestEnv,
-        test_env_api::{HasPublicApiUrl, IcNodeContainer},
+        test_env_api::{HasPublicApiUrl, HasTopologySnapshot, IcNodeContainer, SubnetSnapshot},
     },
     util::{assert_create_agent, block_on},
 };
 use slog::{debug, Logger};
 
 use super::lib::NeuronDetails;
 
+pub fn subnet_sys(env: &TestEnv) -> SubnetSnapshot {
+    env.topology_snapshot()
+        .subnets()
+        .find(|s| s.subnet_type() == SubnetType::System)
+        .unwrap()
+}
+
```

Comment:
nit: I don't think this needs to be `pub`, since from what I could tell, it's (currently) only called from `create_agent` in this same file (so maybe it could even be inlined). Since Nikolas is anyway refactoring these rosetta tests to use pocket IC, I don't think it's worth it for you to spend too much time here though, as long as CI is green!

Reply:
Turns out this function was not needed. Removed.